web/src/app/app.component.html

```diff
@@ -163,13 +163,13 @@
           -->
           <div>
             <div (click)="showDoorbell()" class="footer__button">
-              <mat-icon svgIcon="feedback_toolbar_icon"></mat-icon>
+              <mat-icon class="material-icons-outlined orange-color">feedback</mat-icon>
               Feedback
             </div>
           </div>
         </div>
         <div fxLayout="row" fxLayoutAlign="center center">
-          <mat-icon [style.width.px]="120" svgIcon="assets:dashboardhub"></mat-icon>
+          <img src="assets/icons/logo-horizontal-std.svg" [style.width.px]="120"/>
```

Comment:
Please add a space before the closing of the tag in all places

```html
<img src="assets/icons/logo-horizontal-std.svg" [style.width.px]="120" />
```

Reply:
okay